### ISSUE TITLE
Ethereum Analyzer

### ISSUE THREAD

**Issue by pitasi** [warden-protocol/wardenprotocol#266]
Once #262 is done, use the platform to implement the Ethereum Metadata Provider. It should be able to provide the following data for Ethereum transactions:

- recipient address (to)
~~- tokens being transferred (including ETH **and** ERC-20 tokens)~~ (tracked in separate issue)

This already will be enough for users to write "allowlist" intents, where a transaction can only send tokens to a specific address or can only call a specific Ethereum smart contract (e.g. uniswap).

Once this is done we can remove all the Ethereum-specific code from wardend.
